r2angr/angr

```diff
@@ -65,14 +73,11 @@ For documentation, please read http://z3prover.github.io/api/html/z3.html
 
 In the event of technical difficulties related to configuration, compilation, or installation, please submit issues to https://github.com/z3prover/z3.git
 EOL
-        # Re-Check if z3-solver is installed or not
-        if [ -z "$(pip show z3-solver)" ]; then
-            printf "\033[0;33mSeems like an error occured during installation!"
-            echo "Please try to install it manually"
-            printf "Without z3-solver, we can't install angr on Termux!\033[0m"
-            echo "You can find the venv folder in $HOME/.local/share/radare2/prefix/bin"
-            exit 1
-        fi
+      # this is due to python forcebly downloading pypi cmake for pypcode
+      # which isnt compatible with building inside termux
+      git clone https://github.com/angr/pypcode.git -b v3.3.0 --depth 1
+      sed -i 's/ "cmake" ]/ ]/g' pypcode/pyproject.toml
+      pip install pypcode/ -v
     fi
     pip install -v angr
```

Comment:
should we pin angr to a specific version?

Reply:
Yes i think that would be good